python/cudf_polars/tests/containers/test_dataframe.py

```diff
@@ -161,3 +163,24 @@ def test_empty_name_roundtrips_overlap():
 def test_empty_name_roundtrips_no_overlap():
     df = pl.LazyFrame({"": [1, 2, 3], "b": [4, 5, 6]})
     assert_gpu_result_equal(df)
+
+
+@pytest.mark.parametrize(
+    "arrow_tbl",
+    [
+        pa.table([]),
+        pa.table({"a": [1, 2, 3], "b": [4, 5, 6], "c": [7, 8, 9]}),
+        pa.table({"a": [1, 2, 3]}),
+        pa.table({"a": [1], "b": [2], "c": [3]}),
+        pa.table({"a": ["a", "bb", "ccc"]}),
+        pa.table({"a": [1, 2, None], "b": [None, 3, 4]}),
+    ],
+)
+def test_serialize(arrow_tbl):
```

Comment:
Or maybe `test_serialize_deserialize`?
```suggestion
def test_serialization_roundtrip(arrow_tbl):
```